src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -605,6 +606,279 @@ def make_default_signatures(pretrained: t.Any) -> ModelSignaturesType:
+
+    config = transformers.AutoConfig.from_pretrained(
+        model_name_or_path,
+        proxies=proxies,
+        revision=revision,
+        force_download=force_download,
+        resume_download=resume_download,
+        **extra_hf_hub_kwargs,
+    )
+    if pretrained_model_class is None:
+        pretrained_model_class = getattr(transformers, config.architectures[0])
+        logger.info(
+            f"pretrained_model_class is not provided, bentoml will create a model with the following pretrained model class {config.architectures[0]}. Available pretrained classes for this model: {config.architectures}."
+        )
+
+    model = None
+    if os.path.isdir(model_name_or_path):
+        src_dir = model_name_or_path
+        if sync_with_hub_version:
+            raise BentoMLException(
+                "Cannot sync version with huggingface hub when importing a local model"
+            )
+    else:
+        try:
+            if clone_repository:
+                from huggingface_hub import snapshot_download
+
+                src_dir = snapshot_download(
+                    model_name_or_path,
+                    proxies=proxies,
+                    revision=revision,
+                    force_download=force_download,
+                    resume_download=resume_download,
+                    **extra_hf_hub_kwargs,
+                )
+            else:
+                from transformers.utils import cached_file
+
+                from .utils.transformers import init_empty_weights
+
+                with init_empty_weights():
+                    model = pretrained_model_class.from_pretrained(
+                        model_name_or_path,
+                        proxies=proxies,
+                        revision=revision,
+                        force_download=force_download,
+                        resume_download=resume_download,
+                        **extra_hf_hub_kwargs,
+                    )
+                    path_to_config = cached_file(
+                        model_name_or_path,
+                        CONFIG_JSON_FILE_NAME,
+                        proxies=proxies,
+                        revision=revision,
+                        force_download=force_download,
+                        resume_download=resume_download,
+                        **extra_hf_hub_kwargs,
+                    )
+                    src_dir = os.path.dirname(path_to_config)
+
+                if sync_with_hub_version:
+                    from huggingface_hub.file_download import REGEX_COMMIT_HASH
+
+                    version = extract_commit_hash(src_dir, REGEX_COMMIT_HASH)
+                    if version is not None:
+                        tag.version = version
+        except Exception:
+            logger.info(
+                "Failed to download model file for (%s).",
+                model_name_or_path,
+            )
+            raise
+
+    if model is None:
+        from .utils.transformers import init_empty_weights
```

Comment:
Why there are two different import on L817 and L781 here? Shouldn't this just be on the top of the files?

Reply:
oops, I think this was here because we had a check for accelerate availability before. Since we no longer depend on accelerate, I will move this up to the top.